docs/application/web/guides/data/file-system.md

```diff
@@ -46,15 +45,15 @@ To use the Filesystem API (in [mobile](../../../../org.tizen.web.apireference/ht
 
 ## Managing File Storages
 
-You can manage different storages on the device with the `FileSystemManager` interface (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/filesystem.html#FileSystemManager), [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/filesystem.html#FileSystemManager), and [TV](../../../../org.tizen.web.apireference/html/device_api/tv/tizen/filesystem.html#FileSystemManager) applications).
+You can manage different storages on the device with the `FileSystemManager` interface (in [mobile](../../api/latest/device_api/mobile/tizen/filesystem.html#FileSystemManager), [wearable](../../api/latest/device_api/wearable/tizen/filesystem.html#FileSystemManager), and [TV](../../api/latest/device_api/tv/tizen/filesystem.html#FileSystemManager) applications).
 
 You can retrieve additional information about the storages, including listing available storages and receiving storage change notifications with the `listStorages()` and `addStorageStateChangeListener()` methods provided by the `FileSystemManager` interface.
 
 To manage file storages:
 
 1. To list available storages, use the `listStorages()` method of the `FileSystemManager` interface to search for the storages available on the device.
```

Comment:
Suggestion:

Please change the ordered list under "To manage file storages:" to unordered list (bullets) as it is not a sequential or a procedural step.

Reply:
Done.